sys-config/ltoize/files/package.cflags/ltoworkarounds.conf

```diff
@@ -195,14 +252,20 @@ dev-util/kbuild *FLAGS+="-fuse-ld=bfd"
 
 # BEGIN: -fno-common workarounds
 app-accessibility/at-spi2-atk *FLAGS-="${NOCOMMON}"
+app-admin/logrotate *FLAGS-="${NOCOMMON}"
```

Comment:
I would accept all of these NOCOMMON workarounds in a separate PR if you were willing to do that